monkey/infection_monkey/exploit/hadoop.py

```diff
@@ -40,11 +40,19 @@ def _exploit_host(self):
         urls = self.build_potential_urls(self.host.ip_addr, self.HADOOP_PORTS)
         self.add_vulnerable_urls(urls, True)
         if not self.vulnerable_urls:
+            self.publish_exploitation_event(
+                target=self.host.ip_addr,
+                exploitation_success=False,
+            )
```

Comment:
Add an error message? I realize we're not doing it for `exploit_result`, but maybe do it there too.